src/test/java/mercadopago/client/paymentmethod/PaymentMethodClientTest.java

```diff
@@ -0,0 +1,78 @@
+package mercadopago.client.paymentmethod;
+
+import static com.mercadopago.net.HttpStatus.OK;
+import static mercadopago.helper.MockHelper.generateHttpResponseFromFile;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doReturn;
+
+import com.mercadopago.client.paymentmethod.PaymentMethodClient;
+import com.mercadopago.exceptions.MPException;
+import com.mercadopago.net.MPResourceList;
+import com.mercadopago.resources.paymentmethod.PaymentMethod;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import mercadopago.BaseClientTest;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpRequestBase;
+import org.apache.http.protocol.HttpContext;
+import org.junit.jupiter.api.Test;
+
+class PaymentMethodClientTest extends BaseClientTest {
+
+  private static final String PAYMENT_METHOD_BASE_JSON = "paymentmethod/payment_method_base.json";
+
+  private static final String THUMBNAIL =
+      "https://www.mercadopago.com/org-img/MP3/API/logos/debmaster.gif";
+
+  private static final Long ACCREDITATION_TIME = 1440L;
+
+  private final PaymentMethodClient client = new PaymentMethodClient();
+
+  @Test
+  void list() throws MPException, IOException {
```

Comment:
```suggestion
  void listSuccess() throws MPException, IOException {
```
Sugestão só pra manter um padrão

Reply:
seria bom incuir também um teste de `list` recebendo `MPRequestOptions`